### ISSUE TITLE
Tags and skills return data changed

### ISSUE THREAD

**Issue by itzomen** [divanov11/Mumble#402]
### Is your feature request related to a problem? Please describe.

The return data of the tags and skills is now serialized

```json
# data presently returned
"tags": [
{
"name": "tag1"
},
{
"name": "tag2"
}
```
#

### Describe the solution you'd like

I will be modifying the the mapping to adapt to the change

**Comment by MidouWebDev** [comment]
@itzomen please refer to the contributing guidelines before creating a PR or an issue.
You always forget to add them to the project board !!

https://github.com/divanov11/Mumble/blob/master/Project_Board.md

**Comment by itzomen** [comment]
> @itzomen please refer to the contributing guidelines before creating a PR or an issue.
> You always forget to add them to the project board !!
> 
> https://github.com/divanov11/Mumble/blob/master/Project_Board.md

Oh Sorry, it always slip off my mind.

**Comment by codyseibert** [comment]
I think we are going to go forward with this schema.  The UI fix is here #409
